test-project/Assets/EditmodeTests/Improbable.Gdk.EditModeTests.asmdef

```diff
@@ -1,6 +1,7 @@
 {
     "name": "Improbable.Gdk.EditModeTests",
     "references": [
+        "Improbable.Gdk.QueryBasedInterestHelper",
```

Comment:
At the risk of starting a bikeshed on naming, isn't the `Helper` suffix a bit redundant?

Reply:
Don't worry - everyone loves a good bikeshed 😉 

I think this was to differentiate from QBI itself.

Reply:
^yup what Jamie said

I discussed this with Laura when naming this, and this name gets the point across that you don't _need_ this module to use QBI